Software/Python/grovepi.py

```diff
@@ -329,6 +330,26 @@ def dht(pin, module_type):
 	else:
 		return [float('nan'),float('nan')]
 
+# after a list of numerical values is provided
+# the function returns a list with the outlier(or extreme) values removed
+# make the std_factor_threshold bigger so that filtering becomes less strict
+# and make the std_factor_threshold smaller to get the opposite
+def statisticalNoiseReduction(values, std_factor_threshold = 2):
```

Comment:
@RobertLucian : any reasons why this function is in the grovepi.py main python library